src/amo/utils/server.js

```diff
@@ -59,3 +68,22 @@ export const viewFrontendVersionHandler = ({
     });
   };
 };
+
+export const viewHeartbeatHandler = ({
+  _config = config,
+  _fetch = fetch,
+}: ViewHeartbeatHandlerParams = {}): ExpressHandler => {
+  const apiURL = `${_config.get('apiHost')}${_config.get(
+    'apiPath',
+  )}${_config.get('apiVersion')}/site/?disable_caching`;
+
+  return async (req: typeof $Request, res: typeof $Response) => {
+    try {
+      const response = await _fetch(apiURL);
+      const ok = response.status === 200;
+      res.status(ok ? response.status : 500).end(ok ? 'ok' : 'ko');
+    } catch (err) {
+      res.status(500).end();
```

Comment:
Do we want to return `ko` here as well?

```suggestion
      res.status(500).end('ko');
```